### DISCUSSION TITLE
how does daily discharge relate to total cell runoff

### DISCUSSION THREAD

**Discussion by diegodinamarca** [mhm-ufz/mHM#60] [Q&A] [answered]
Hello mHM dev team

I'm running mHM for a basin in Chile testing different land covers as forcing datasets. I was wondering how the results shown at the daily_discharge.out file, that has the Qobs and Qsim at the discharge point of the catchment, relate to the Total discharge generated per cell (L1_total_runoff) that is spatially distributed. Is it possible to obtain the Qsim by applying some function to the spatially distributed output?

Thank you for your help. Kind regards.

**Comment by rohinikumar82** [comment, marked as answer]
After the "Total discharge" generated from every grid cell - it is routed following the stream network within the mHM code (via mRM). The model code already apply and do for you the "some function to the spatially distributed output"  to get "Qsim". Stream routing is based on the Muskimgum method  and the code-base apply the novel upscaling approach to specify river network at the choice of your modelling resolution.

**Reply by diegodinamarca** [reply]
ty for your reply!
